src/test/regress/expected/multi_schema_support.out

```diff
@@ -1090,6 +1090,113 @@ SELECT sum(result::int) FROM run_command_on_shards('run_test_schema.test_table',
  24576
 (1 row)
 
+-- test capital letters on both table and schema names
+SET citus.task_executor_type to "real-time";
+-- create schema with weird names
+CREATE SCHEMA "CiTuS.TeeN";
+CREATE SCHEMA "CiTUS.TEEN2";
+-- create table with weird names
+CREATE TABLE "CiTuS.TeeN"."TeeNTabLE.1!?!"(id int, "TeNANt_Id" int);
+CREATE TABLE "CiTUS.TEEN2"."CAPITAL_TABLE"(i int, j int);
+-- create distributed table with weird names
+SELECT create_distributed_table('"CiTuS.TeeN"."TeeNTabLE.1!?!"', 'TeNANt_Id');
+ create_distributed_table 
+--------------------------
+ 
+(1 row)
+
+SELECT create_distributed_table('"CiTUS.TEEN2"."CAPITAL_TABLE"', 'i');
+ create_distributed_table 
+--------------------------
+ 
+(1 row)
+
+-- insert into table with weird names
+INSERT INTO "CiTuS.TeeN"."TeeNTabLE.1!?!" VALUES(1, 1),(1, 0),(0, 1),(2, 3),(3, 2),(4, 4);
+INSERT INTO "CiTUS.TEEN2"."CAPITAL_TABLE" VALUES(0, 1),(1, 0),(2, 1),(4, 3),(3, 2),(4, 4);
+-- join on tables with weird names
+SELECT * 
+FROM "CiTuS.TeeN"."TeeNTabLE.1!?!", "CiTUS.TEEN2"."CAPITAL_TABLE" 
+WHERE "CiTUS.TEEN2"."CAPITAL_TABLE".i = "CiTuS.TeeN"."TeeNTabLE.1!?!"."TeNANt_Id"
+ORDER BY 1,2,3,4;
+ id | TeNANt_Id | i | j 
+----+-----------+---+---
+  0 |         1 | 1 | 0
+  1 |         0 | 0 | 1
+  1 |         1 | 1 | 0
+  2 |         3 | 3 | 2
+  3 |         2 | 2 | 1
+  4 |         4 | 4 | 3
+  4 |         4 | 4 | 4
+(7 rows)
+
+-- add group by, having, order by clauses
+SELECT * 
```

Comment:
I think this all good, we don't need any extra tests. But, we could maybe execute one of the tests when `set search_path = "CiTUS.TEEN2"` and not adding `"CiTUS.TEEN2"` to the query. This is just for a sanity check.